test/show.jl

```diff
@@ -104,7 +119,12 @@ end
     @test ColorTypes.colorant_string_with_eltype(RGB{Union{}}) == "RGB{Union{}}"
     @test ColorTypes.colorant_string_with_eltype(RGB{<:Fractional}) == "RGB"
     @test ColorTypes.colorant_string_with_eltype(HSV{<:AbstractFloat}) == "HSV"
-    @test ColorTypes.colorant_string_with_eltype(RGB{Fractional}) == "RGB{Union{AbstractFloat, FixedPoint}}"
+    rgb_fractional = ColorTypes.colorant_string_with_eltype(RGB{Fractional})
+    if VERSION >= v"1.6.0-DEV.356"
+        @test eval(Meta.parse(rgb_fractional)) == RGB{Union{AbstractFloat, FixedPoint}}
+    else
+        @test rgb_fractional == "RGB{Union{AbstractFloat, FixedPoint}}"
```

Comment:
and `@test rgb_fractional == "RGB{$(Fractional)}"`, too?

Reply:
Unlike the cases of `summary`, this test aims to clarify the current behavior. The important thing is that it is "not" `"RGB{Fractional}"`.
In this `Fractional` case, the results on the nightly build is still fragile, so I decided to check for the minimal round-tripness.